staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/webhook.go

```diff
@@ -152,6 +159,18 @@ func (a *Webhook) ShouldCallHook(h webhook.WebhookAccessor, attr admission.Attri
 	if !matches && matchObjErr == nil {
 		return nil, nil
 	}
+	decisions, err := a.evaluateCELRules(h, attr, o)
+	if err != nil {
+		//TODO: what to error handle here? fall back to fail policy?
+		return nil, nil
+	}
+	for _, decision := range decisions {
+		klog.Infof("ivelichkovich decision action is %v", decision.Action)
+		klog.V(5).Infof("Did not call webhook due to match condition %v", decision.Message)
+		if decision.Action == cel.ActionDeny {
+			return nil, nil
```

Comment:
TODO:
- Observe metric (see https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/3716-admission-webhook-match-conditions/README.md#monitoring-requirements)
- Log it at high-verbosity (maybe above `V(5)`?)